### ISSUE TITLE
compute_metrics would first normalize embedding ?

### ISSUE THREAD

**Issue by PapaMadeleine2022** [OML-Team/open-metric-learning#345]
Hello, I use the follow code:

```
val_loader = torch.utils.data.DataLoader(val_dataset, batch_size=512)
calculator = EmbeddingMetrics()
calculator.setup(num_samples=len(val_dataset))

with torch.no_grad():
    for batch in tqdm(val_loader):
        batch["embeddings"] = model(batch["input_tensors"])
        calculator.update_data(batch)

metrics = calculator.compute_metrics()
```

In default, `embeddings` is not normalized by the model,  so `compute_metrics` would first normalize `embeddings`? I checked the code, and it seems that no `normalize` function in function `compute_metrics`.

**Comment by AlekseySh** [comment]
@PapaMadeleine2022 It's true. The normalization only happens in the model. In metrics we compute L2 (euclidian distance) on the given vectors doesn't matter if they are normalized or not.

**Comment by PapaMadeleine2022** [comment]
@PapaMadeleine2022 In my opinion, `compute_metrics` for normalized `embeddings` can get better result than the `embeddings` not normalized?

**Comment by AlekseySh** [comment]
@PapaMadeleine2022 Depends on how you trained your model. If you normalized your features during the training, I would recommend also normalising them during the validation. But sometimes you can benefit from normalizing features for validation, even if it was not the case in the train.
